test/db/esil/mips_32

```diff
@@ -1014,16 +1014,12 @@ ar v0=0x00112233
 aeip
 aes
 ar s0
-ar hi
-ar lo
```

Comment:
《MIPS® Architecture For Programmers Volume II-A: The MIPS64® Instruction Set, Revision 3.02》P248
This is how the MUL instruction is described：
The 32-bit word value in GPR rs is multiplied by the 32-bit value in GPR rt, treating both operands as signed values,
to produce a 64-bit result. The least significant 32 bits of the product are sign-extended and written to GPR rd. **The
contents of HI and LO are UNPREDICTABLE after the operation.** No arithmetic exception occurs under any circumstances.
So we should not test the value of HI、LO

Reply:
Ok then